src/graphql/object_type.cr

```diff
@@ -46,9 +46,10 @@ module GraphQL::ObjectType
             begin
               errors.concat _graphql_resolve(context, selection, json)
             rescue e
-              if !e.message.nil?
+              message = context.handle_exception(e)
+              if !message.nil?
```

Comment:
```suggestion
              if message = context.handle_exception(e)
```

This is a pretty standard Crystal convention, and you wouldn't need to use the `not_nil!` below

Reply:
Nice, thanks